app/parse.py

```diff
@@ -1,5 +1,12 @@
+import csv
 from dataclasses import dataclass
 
+import requests
+from bs4 import BeautifulSoup, Tag
+
+BASE_URL = "https://quotes.toscrape.com/"
+PAGE_URL = f"{BASE_URL}/page/"
```

Comment:
The `PAGE_URL` variable is incorrectly formatted. The extra slash at the end of `BASE_URL` and the beginning of `/page/` will result in URLs like `https://quotes.toscrape.com//page/1`. Consider removing the extra slash from `BASE_URL` or `PAGE_URL`.